### ISSUE TITLE
"Include hashtag when copying" switch doesn't work when turned on with enter key

### ISSUE THREAD

**Issue by edelstone** [edelstone/tints-and-shades#40]
If you tab to the "Include hashtag when copying" switch and activate it using the enter key, it doesn't actually activate the switch. Same with deactivate via tab/enter key. Ideally this should work the same as being activated/deactivated with the mouse.
